### PR TITLE
Front/home footer contact

### PR DESCRIPTION
**Pull request by phindell**
## Check before merging

- [x] Ticket : [(3) ETQU, je vois un footer avec des la section "Parc National, ..."](https://trello.com/c/7XWy4wh7/94-3-etqu-je-vois-un-footer-avec-des-la-section-parc-national)
- [x] I made sure that all displayed texts are imported from translation files.
- [x] I made sure ticket is up to date on Trello.

### Mobile
image
image

### Desktop
image

### CODE REVIEW

File: frontend/src/components/pages/home/components/HomeFooter/HomeFooter.tsx [hunk 1]
```diff
@@ -18,8 +19,9 @@ const socialNetworksIcons: { [key: string]: React.ReactNode } = {
 };
 
 export const HomeFooter: React.FC<HomeFooterProps> = ({ socialNetworks }) => {
+  // TODO déplacer les logos dans un composant à part ?
```
Comment (Weegle99) [inline frontend/src/components/pages/home/components/HomeFooter/HomeFooter.tsx:22]: T'as une reco la dessus ?

Comment (phindell) [reply frontend/src/components/pages/home/components/HomeFooter/HomeFooter.tsx:22]: Pas vraiment :( Le composant n'est pas réuilisé ailleurs et n'a pas de state, la seule raison de le mettre dans un composant à part c'est de "cleaner" notre composant HomeFooter. Toutes les autres sections sont des composants à part

**Review by Weegle99 — COMMENTED** [review]
T'as pu check les edge cases type texte trop long ? Ecran réduit ?

**Review by Weegle99 — COMMENTED** [review]
On peut faire un test d'intégration pour vérifier le comportement du clic sur le + et sur le lien mail
Dispo pour t'aider

**Comment by phindell** [comment]
@Weegle99 pour le responsive le footer s'allonge si le texte s'allonge aussi / si l'écran est diminué. En desktop c'est peu contraignant pour le moment, à rechecker quand tous les blocs seront présents. En mobile j'ai corrigé le shrink et la position de l'icône
image

File: frontend/src/components/pages/home/components/HomeFooter/HomeFooter.tsx (hunk 1 above)
Comment (phindell) [reply frontend/src/components/pages/home/components/HomeFooter/HomeFooter.tsx]: Edit : refacto faite, composant "SocialNetworks" extrait du footer

**Review by Weegle99 — APPROVED** [review]
